### ISSUE TITLE
Issues after updating node to v4.2.1

### ISSUE THREAD

**Issue by Faiz7412** [yeoman/yeoman#1557]
I had lots of generators installed, yo worked perfectly fine. But things changed after updating node version. I updated node version using nvm. Now, my installed generators fail to run. I thought reinstalling Yo might help, but turns out it says 'v4.2.1' version of node as 'outdated'.
![screenshot from 2015-10-31 16 55 58](https://cloud.githubusercontent.com/assets/8874731/10863349/98d71758-7ff0-11e5-9582-1b549d9331b3.png)
Yes I know I made a mistake by not using -g command (I did that on purpose)

**Comment by Faiz7412** [comment]
Also, I tried re-installing meanjs generator as it was not working earlier. It still gives errors
It always says that a particular module is missing. I installed a missing module earlier explicitly, but this goes on and on.
For example: 
![screenshot from 2015-10-31 17 23 01](https://cloud.githubusercontent.com/assets/8874731/10863450/1e8e11b4-7ff4-11e5-8083-6fc665e4b168.png)
And then, 
![screenshot from 2015-10-31 17 23 10](https://cloud.githubusercontent.com/assets/8874731/10863451/1fd128c2-7ff4-11e5-8680-1303d3ac137d.png)
Even uninstalling and reinstalling, or updating doesn't help!
